codenney/fashion_api
# Chore(connect to fashiondb)

### What does this PR do?
* To successfully create a connection with the fashiodb

### Description of the task to be completed?
* Install the dependencies 
* Provide the necessary parameters to connect the db

### How should this be manually tested?
* On the terminal, run the command npm run start:dev to start the server and confirm DB connection
* If successful, a message will be logged on the terminal
